packages/support/src/settings/_variables.color.scss

```diff
@@ -26,19 +26,24 @@ $color-primary-alt-lightest: #e1f3f8 !default;
 
 $color-red: #e31c3d !default;
 $color-red-dark: #cd2026 !default;
+$color-red-darker: #b31e22 !default;
 $color-red-darkest: #981b1e !default;
 $color-red-light: #e59393 !default;
 $color-red-lighter: #f5adb9 !default;
 $color-red-lightest: #f9dede !default;
 
 $color-gold: #fdb81e !default;
+$color-gold-dark: #e4a61b !default;
```

Comment:
just curious, how did u calculate these new values?

Reply:
I took a look at the other palettes and used https://calcolor.co/blend/2e8540/000000 to determine shades that were balanced. 

When I was researching how different shades were generated they did not seem to be consistent. Take a look at the SCSS of my codepen for exploration of how current colors were being calculated. https://codepen.io/line47/pen/povZwmo?editors=0100